gradoop-data-integration/src/main/java/org/gradoop/dataintegration/transformation/InvertEdges.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.dataintegration.transformation;
+
+import org.apache.flink.util.Preconditions;
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.pojo.Edge;
+import org.gradoop.flink.model.api.functions.TransformationFunction;
+
+/**
+ * Each a given edge label each edge gets its label changed + source and target
```

Comment:
Is there a typo? Or my English is not "the yellow from the egg" ^^ Maybe: `An edge transformation that swaps the source and target of an edge with a given label and renames it.`